test/passing/module.ml

```diff
@@ -43,3 +43,11 @@ with type t := t
  and type s := s = struct
   let () = ()
 end
+
+include struct
+  (* a *)
+end
+
+include A (struct
+  (* a *)
+end)
```

Comment:
Shouldn't this be `include A ( (* a *) )` ?

Reply:
I chose to keep the original behavior that is to have the `struct`/`end` keywords if the structure is non-empty, and the structure is not empty since there is at least one comment.

Reply:
I agree. Implementing this is extremely tricky anyway, as the comment inside the parenthesis is placed juste after the lid.

Reply:
And by the way if we omit the keywords, the comment is placed before `( )` and not inside.